### DISCUSSION TITLE
Can jtl-reporter be integrated with K6.io

### DISCUSSION THREAD

**Discussion by santhosh-uc** [ludeknovy/jtl-reporter#178] [General]
I followed how to post a csv file to jtl-reporter by making an API which returned me an itemId. I am not able to see any data displaying on the dashboard. 
Can I make a API call to send data from k6 to jtl-reporter. Is there any wrapper available to do so.

**Comment by ludeknovy** [comment]
Hi @santhosh-uc. Currently, there is no plugin that would allow you to integrate it with k6.

Top of my head I can think of two options here:
1. You could create a K6 plugin, that would send the data to the JtlReporter
2. Maybe try to use [taurus](https://gettaurus.org/) to launch your k6 tests and it should generate kpi.jtl file, that can be uploaded to JtlReporter.

**Reply by santhosh-uc** [reply]
Thank you for the response. I will give it a try

**Reply by ludeknovy** [reply]
Let me know if you made it work.

**Reply by santhosh-uc** [reply]
I checked Taurus.  I need to write all my loads in Taurus and then execute it.This wont work for me,  since I have k6 scripts written for all my API. I need a way to send the test results to JTL-Reporter from my k6 scipt when the test runs. for Locust.io I see there is a JtlListener.  Is there a direct way to send data using API ?.  Is JTlListener internal making API call to send report ?

**Reply by ludeknovy** [reply]
@santhosh-uc 
```
I need to write all my loads in Taurus and then execute it
```
Not sure I understand it, but you can use taurus only an executor: https://gettaurus.org/docs/K6/


If you want to find out how to get data in, please refere to https://jtlreporter.site/docs/integrations/introduction
